rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/JobResultImpl.java

```diff
@@ -54,24 +57,25 @@
 
     private Map<String, TaskResult> preciousResults;
 
+    private Map<String, Serializable> resultMap;
+
     private Map<String, TaskResult> exceptionResults;
 
-    JobResultImpl(JobResultData data) {
+    JobResultImpl(JobResultData data) throws IOException, ClassNotFoundException {
         JobIdData id = data.getId();
         jobId = new JobIdImpl(id.getId(), id.getReadableName());
         allResults = createTaskResultMap(data.getAllResults());
         preciousResults = createTaskResultMap(data.getPreciousResults());
         exceptionResults = createTaskResultMap(data.getExceptionResults());
+        resultMap = ObjectByteConverter.mapOfBase64StringToSerializable(data.getSerializedResultMap());
```

Comment:
Can handle the exception here ? It is better to not throw an exception in the method prototype otherwise every other method that use JobResultImpl will have to handle those exceptions. i.e toJobResult

Reply:
We could eventually throw a runtime exception to avoid handling.